servicebroker/lib/index.js

```diff
@@ -14,6 +15,28 @@ var app = express();
 app.use(bodyParser.urlencoded({ extended: true }));
 app.use(bodyParser.json());
 
+var auth = function (req, res, next) {
+  
+  function unauthorized(res) {
+    res.set('WWW-Authenticate', 'Basic realm=Authorization Required');
```

Comment:
That isn't a realm. A realm must be a quoted string. (https://tools.ietf.org/html/rfc2617)